packages/jaeger-ui/src/utils/helpers/ReactShallowRenderer.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2023 The Jaeger Authors
+// Adopted from https://thesametech.com/snapshot-testing-in-rtl/
+// Copyright (c) 2023 Ildar Sharafeev
```

Comment:
Author's permission in this comment: https://github.com/jaegertracing/jaeger-ui/pull/1653#discussion_r1292021157